packages/clay-management-toolbar/src/ResultsBar.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayLabel from '@clayui/label';
+import ClayLink from '@clayui/link';
+import React from 'react';
+import ResultsBarItem from './ResultsBarItem';
+
+interface IProps {
+	clearLabel?: string;
+	clearResultsURL?: string;
+	filterLabels?: Array<
+		React.ComponentProps<typeof ClayLabel> & {value: React.ReactText}
+	>;
+	onClearButtonClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	searchValue?: string;
+	spritemap?: string;
+	totalItems?: number;
+}
+
+const ResultsBar: React.FunctionComponent<IProps> = ({
+	clearLabel,
+	clearResultsURL,
+	filterLabels = [],
+	onClearButtonClick,
+	searchValue,
+	spritemap,
+	totalItems,
+}) => (
+	<nav className="tbar tbar-inline-xs-down subnav-tbar subnav-tbar-primary">
+		<div className="container-fluid container-fluid-max-xl">
+			<ul className="tbar-nav tbar-nav-wrap">
+				<ResultsBarItem expand={!(filterLabels.length > 0)}>
+					<span className="component-text text-truncate-inline">
+						<span className="text-truncate">
+							{`${totalItems} results for "`}
+							<strong>{searchValue}</strong>
+							{'"'}
+						</span>
+					</span>
+				</ResultsBarItem>
+				{filterLabels.map(({value, ...others}, index: number) => (
+					<ResultsBarItem
+						expand={filterLabels.length - 1 === index}
+						key={index}
+					>
+						<ClayLabel
+							{...others}
+							className="component-label tbar-label"
+							displayType="unstyled"
+							spritemap={spritemap}
+						>
+							{value}
+						</ClayLabel>
+					</ResultsBarItem>
+				))}
+				<ResultsBarItem>
+					{clearResultsURL ? (
```

Comment:
We tend to do this `Link || Button` logic quite a bit. I wonder if we could create some conditional component that could handle the logic for us.

Reply:
I think that makes sense, it is likely that we will use more and many other places. But I think it would be better to add it to `shared` and not expose it.